content/webapp/components/EventsByMonth/EventsByMonth.tsx

```diff
@@ -49,29 +52,29 @@ const EventsByMonth: FunctionComponent<Props> = ({ events, links }) => {
                 id="monthControls"
                 activeId={activeId}
                 setActiveId={setActiveId}
-                items={groups}
+                items={monthsWithEvents}
               />
             </div>
           </div>
         </CssGridContainer>
       </Space>
 
-      {groups
-        .filter(g => activeId === g.id)
-        .map(g => (
+      {monthsWithEvents
+        .filter(({ id }) => activeId === id)
+        .map(({ id, month, events }) => (
           <div
-            key={g.id}
+            key={id}
             className={cssGrid(gridSize12)}
             style={{ display: 'block' }}
           >
-            <h2 className="container" id={g.id}>
-              {g.month.month}
+            <h2 className="container is-hidden" id={id}>
```

Comment:
This got lost in a refactor -- previously it was being added with the condition `Boolean(activeId)`, which always evaluates to true, but I misread it as always evaluating to false.

The practical effect is that clicking on the tabs now causes the page to move around whenever you pick a month on the What's On page.